wolfssl/wolfcrypt/oid_sum.h

```diff
@@ -26,6 +26,10 @@
 #ifndef WOLF_CRYPT_OID_SUM_H
 #define WOLF_CRYPT_OID_SUM_H
 
+/* Note for some CPUs smaller than 32 bit, the upper 16 bits of new OID
```

Comment:
This is because the enum type int is 16-bit and it cannot hold the 32-bits so the compiler truncates the upper bits.